data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -428,22 +390,6 @@ def _pollution_factor(self) -> bool:
             >= self.ENVIRONMENTAL_BURDEN_THRESHOLD
         )
 
-        air_toxics_cancer_risk_threshold = (
```

Comment:
@lucasmbrown-usds I am a bit perplexed - an earlier PR had this added. I just wanted to confirm this still to be removed?

Reply:
Yes, these are being intentionally removed.